modules/navier_stokes/src/postprocessors/InternalVolumetricFlowRate.C

```diff
@@ -0,0 +1,162 @@
+            ? _vel_y[_qp]
+            : MetaPhysicL::raw_value(_fv_vel_y->getInternalFaceValue(neighbor, *fi, _vel_y[_qp]));
+
+    const auto & vz_face =
+        !getFieldVar("vel_z", 0)
+            ? _vel_z[_qp]
+            : MetaPhysicL::raw_value(_fv_vel_z->getInternalFaceValue(neighbor, *fi, _vel_z[_qp]));
+
+    /// Compute the advected quantity on the face
+    Real advected_quantity;
+    if (parameters().isParamSetByUser("advected_variable"))
+    {
+      // If user did not request an interpolation method, use what the kernels are
+      // most likely to use
+      if (!parameters().isParamSetByUser("advected_interp_method"))
+      {
+        advected_quantity =
+            !getFieldVar("advected_variable", 0)
+                ? _advected_variable[_qp]
+                : MetaPhysicL::raw_value(_fv_advected_variable->getInternalFaceValue(
+                      neighbor, *fi, _advected_variable[_qp]));
+      }
+      else
+      {
+        /// Get neighbor value
+        const auto & advected_variable_neighbor =
+            !getFieldVar("advected_variable", 0)
+                ? _advected_variable[_qp]
+                : MetaPhysicL::raw_value(_fv_advected_variable->getNeighborValue(
+                      neighbor, *fi, _advected_variable[_qp]));
+
+        Moose::FV::interpolate(_advected_interp_method,
+                               advected_quantity,
+                               MetaPhysicL::raw_value(_advected_variable[_qp]),
+                               MetaPhysicL::raw_value(advected_variable_neighbor),
+                               RealVectorValue(vx_face, vy_face, vz_face),
+                               *fi,
+                               true);
+      }
+    }
+    else if (parameters().isParamSetByUser("advected_mat_prop"))
+    {
+      /// The material property needs to be interpolated since we are on an internal face
+      Moose::FV::interpolate(_advected_interp_method,
+                             advected_quantity,
+                             MetaPhysicL::raw_value(_advected_material_property[_qp]),
+                             MetaPhysicL::raw_value(_advected_material_property_neighbor[_qp]),
+                             RealVectorValue(vx_face, vy_face, vz_face),
+                             *fi,
+                             true);
+    }
+    else
+      advected_quantity = 1;
+
+    return advected_quantity * RealVectorValue(vx_face, vy_face, vz_face) * _normals[_qp];
+  }
+  else
+#endif
+      if (parameters().isParamSetByUser("advected_variable"))
+    return _advected_variable[_qp] * RealVectorValue(_vel_x[_qp], _vel_y[_qp], _vel_z[_qp]) *
+           _normals[_qp];
+  else if (parameters().isParamSetByUser("advected_mat_prop"))
+    return MetaPhysicL::raw_value(_advected_material_property[_qp]) *
+           RealVectorValue(_vel_x[_qp], _vel_y[_qp], _vel_z[_qp]) * _normals[_qp];
+  else
+    return RealVectorValue(_vel_x[_qp], _vel_y[_qp], _vel_z[_qp]) * _normals[_qp];
```

Comment:
These should all be finite element cases right? If so, I think we should wrap this whole section in braces. That should make sense regardless of whether `MOOSE_GLOBAL_AD_INDEXING` is defined, I think.

Reply:
yup.
Good call this fixes the clang formatting abuse of power